packages/interfaces/src/service/state-service.interface.ts

```diff
@@ -31,6 +31,11 @@ export interface StateOptions {
      * The functions will be bound to the web component instance
      */
     filter?: Array<(state: any) => boolean>;
+
+    /**
+     * A component to filter any changes to the state by related limetype.
```

Comment:
This is way too specific :slightly_smiling_face: Something like this would be better I think
```suggestion
     * The component that is subscribing to the state changes
```